src/main/java/com/booleanuk/core/Exercise.java

```diff
@@ -1,73 +1,73 @@
 package com.booleanuk.core;
 import com.booleanuk.helpers.ExerciseBase;
 
-public class Exercise extends ExerciseBase {
+public class Exercise extends ExerciseBase  {
     public int numOne = 8;
     public int numTwo = 16;
     public int numThree = 32;
 
 
     // 1. Change the value of the member below to be the result of adding numOne and numTwo together
-    public int numOnePlusTwo = 0;
+    public int numOnePlusTwo = numTwo + numOne;
 
 
     // 2. Change the value of the member below to be the result of multiplying numThree by numTwo
-    public int numThreeTimesNumTwo = 0;
+    public int numThreeTimesNumTwo = numThree*numTwo;
```

Comment:
You've mostly used spaces between variables eg: `numTwo + numOne`. 

It's best to keep things consistent, spaces are good for readability in general, just be sure to be consistent.